walkthroughs/howto-access-log/README.md

```diff
@@ -0,0 +1,174 @@
+```
+
+## Step 3: Update logging format and Verify
+
+Our next step is to update the logging format and test it out.
+
+1. check meshes is deployed
+
+```bash
+aws --endpoint-url https://frontend.$AWS_DEFAULT_REGION.prod.lattice.aws.a2z.com --region $AWS_DEFAULT_REGION appmesh list-meshes
+```
+2. check virtual node green (on fargate)
+
+```bash
+aws --endpoint-url https://frontend.$AWS_DEFAULT_REGION.prod.lattice.aws.a2z.com --region $AWS_DEFAULT_REGION appmesh describe-virtual-node --virtual-node-name colorteller-green-vn --mesh-name $MESH_NAME
+```
+
+3. update virtual node with text logging format in it
+
+```bash
+aws --endpoint-url https://frontend.$AWS_DEFAULT_REGION.prod.lattice.aws.a2z.com --region $AWS_DEFAULT_REGION appmesh update-virtual-node --virtual-node-name colorteller-blue-vn --mesh-name $MESH_NAME --cli-input-json file://src/blue-text-format.json
+aws --endpoint-url https://frontend.$AWS_DEFAULT_REGION.prod.lattice.aws.a2z.com --region $AWS_DEFAULT_REGION appmesh update-virtual-node --virtual-node-name colorteller-green-vn --mesh-name $MESH_NAME --cli-input-json file://src/green-text-format.json
+```
+If you go to ECS -> Green/Blue backend task -> log -> envoy and search for Green/Blue
+You should see logs like this:
+
+```
+BlueTestLog::200:path=/ping
+```
+
+4. update virtual node with json logging format in it
+
+```bash
+aws --endpoint-url https://frontend.$AWS_DEFAULT_REGION.prod.lattice.aws.a2z.com --region $AWS_DEFAULT_REGION appmesh update-virtual-node --virtual-node-name colorteller-blue-vn --mesh-name $MESH_NAME --cli-input-json file://src/blue-json-format.json
+aws --endpoint-url https://frontend.$AWS_DEFAULT_REGION.prod.lattice.aws.a2z.com --region $AWS_DEFAULT_REGION appmesh update-virtual-node --virtual-node-name colorteller-green-vn --mesh-name $MESH_NAME --cli-input-json file://src/green-json-format.json
+```
+
+If you go to ECS -> Green/Blue backend task -> log -> envoy and search for Green/Blue
+You should see logs like this:
+
+```
+{"BlueTestLog":200,"protocol":"HTTP/1.1"}
+```
+
+5. update virtual node with no logging format in it
+
+```bash
+aws --endpoint-url https://frontend.$AWS_DEFAULT_REGION.prod.lattice.aws.a2z.com --region $AWS_DEFAULT_REGION appmesh update-virtual-node --virtual-node-name colorteller-blue-vn --mesh-name $MESH_NAME --cli-input-json file://src/blue-no-format.json
+aws --endpoint-url https://frontend.$AWS_DEFAULT_REGION.prod.lattice.aws.a2z.com --region $AWS_DEFAULT_REGION appmesh update-virtual-node --virtual-node-name colorteller-green-vn --mesh-name $MESH_NAME --cli-input-json file://src/green-no-format.json
+```
+Search for `colorteller-blue.logging.local` in console you and should see the default format like following
+```
+[2022-08-04T23:00:29.383Z] "GET /ping HTTP/1.1" 200 - 0 0 1 0 "-" "Envoy/HC" "05ba7454-fce2-9360-92e3-a4da997d4f44" "colorteller-blue.logging.local:9080" "127.0.0.1:9080"
+```
+6. change between different formats or change the pattern to find bugs!
```

Comment:
This probably doesn't belong in customer facing stuff.

Reply:
yeah